README.md

```diff
@@ -1,6 +1,6 @@
 # SpingQuick
```

Comment:
We can remove "# SpingQuick" and just keep "# QuickBytesOnSpring" for a single title

Reply:
Resolved